nixos/modules/system/boot/plymouth.nix

```
@@ -97,16 +96,17 @@ in
       logo = mkOption {
         type = types.path;
         # Dimensions are 48x48 to match GDM logo
-        default = "${nixos-icons}/share/icons/hicolor/48x48/apps/nix-snowflake-white.png";
-        defaultText = literalExpression ''"''${nixos-icons}/share/icons/hicolor/48x48/apps/nix-snowflake-white.png"'';
+        default = "${pkgs.nixos-icons}/share/icons/hicolor/48x48/apps/nix-snowflake-white.png";
+        defaultText = literalExpression ''"''${pkgs.nixos-icons}/share/icons/hicolor/48x48/apps/nix-snowflake-white.png"'';
         example = literalExpression ''
           pkgs.fetchurl {
             url = "https://nixos.org/logo/nixos-hires.png";
             sha256 = "1ivzgd7iz0i06y36p8m5w48fd8pjqwxhdaavc0pxs7w1g7mcy5si";
           }
         '';
         description = lib.mdDoc ''
-          Logo which is displayed on the splash screen.
+          Logo which is displayed on the splash screen. Currently
+          supports PNG file format only.
```

```suggestion
          Logo which is displayed on the splash screen.
          Currently supports PNG file format only.
```
lets make this easy readable